### DISCUSSION TITLE
How to model diffusion of electron-hole pairs from energy deposition?

### DISCUSSION THREAD

**Discussion by Jaap1957** [nrc-cnrc/EGSnrc#867] [Questions]
Dear Forum,

I try to model a pixelated detector (camera).
Energy deposited is converted to e-h pairs at 3.68eV per pair. Per energy deposition that counts to ~1000 eh pairs. Many of these recombine; Each e- of a pair diffuses towards the + voltage with some gaussian diffusion law.
Through this mechanism part of an energy deposition goes to an adjacent pixel, and pixel signal flattens.

How would you handle this?
Can I add such an "eh shower" to the energy depositions and score the eh numbers passed to the pixels?
Can I use EGSnrc logic to calculate the distribution of eh numbers?
The calculation has some resemblance to what happens in EGSnrc. 

This is quite a general problem, also encountered in scintillators where photons are formed.

Thanks !
Jaap

**Comment by ftessier** [comment]
There is no modelling of electric transport in EGSnrc, to model what happens after energy has been deposited in the material, phenomena such as drift, recombination, cascades from further ionizations below the 1 keV thershold (the lowest energy cutoff in EGSnrc), etc. This is something you would have to model yourself, either inside EGSnrc (this has been done before, for example in `egs_pet`, unreleased, see #807), or using an external custom or commercial software which models electronic transport. You could start by converting individual energy deposition events in EGSnrc into equivalent electron-hole pairs locally at the location of each event.
